### DISCUSSION TITLE
用这个，我还不如去踩bevy的坑来得爽。

### DISCUSSION THREAD

**Discussion by examango** [ejoy/ant#80] [General]
来自于一个玩《三国志战棋版》很不爽的用户。

**Comment by examango** [comment]
还有，阿里系的开源产品，名声在外，我有点害怕。

信云风，但不信阿里系。

**Comment by examango** [comment]
一个正经的问题，有没有解析并播放spine骨骼动画的现成模块？

**Comment by julywind168** [comment]
bevy ecs不错的😌 非常有潜力

**Reply by lemospring** [reply]
你用过嘛，就说非常有潜力

**Comment by examango** [comment]
今天看了下，《三国志战棋版》公众号的充值入口关了。

我去找苹果反馈的问题，成效不错。
